packages/shared/src/components/profile/UserMetadata.tsx

```diff
@@ -66,8 +70,9 @@ export function UserMetadata({
           date={new Date(createdAt)}
         />
       </div>
+      {isPlus && showPlusSubscription && <PlusUser />}
```

Comment:
Doesn't the showPlus already handle isPlus @omBratteng

Reply:
This component is used two places. One for when viewing profiles, and one for profile menu.
So in this case, `isPlus` is for the user you're viewing / your profile menu. And `showPlusSubscription` is if you're enrolled.